web/src/containers/TopNav.js

```diff
@@ -24,15 +31,25 @@ class TopNav extends Component {
         <div className="sm-col-right h5">
           {authenticated ? (
             <div>
-              <Link to="/state-start" className="btn">
-                Get started
-              </Link>
-              <button type="button" className="btn" onClick={this.handleLogout}>
-                Log out
+              <button type="button" className="btn h5 regular">
+                <span className="mr-tiny">Notifications</span>
+                <FontAwesomeIcon icon={faBell} />
+              </button>
+              <button type="button" className="btn h5 regular">
+                <span className="mr-tiny">Help</span>
+                <FontAwesomeIcon icon={faHelp} />
+              </button>
```

Comment:
I'm curious if the help button should really only be accessible if you're logged in.  Maybe a question for @lauraponce?  If it should always be available, is it okay to create an issue to fix it so we can go ahead and merge this?